### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hamelsmu**
Added two new Machine Learning Ops references

cc: @neovintage @awmatheson

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,6 +451,8 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Query Experiment Tracking Results From Weights & Biases](https://github.com/machine-learning-apps/wandb-action)
 - [Run Parameterized Jupyter Notebooks](https://github.com/yaananth/run-notebook)
 - [Compile, Deploy and Run Kubeflow Pipeline](https://github.com/NikeNano/kubeflow-github-action)
+- [Automatically Dockerize a data-science repository as a Jupyter Server](https://github.com/machine-learning-apps/repo2docker-action)
+- [Azure Machine Learning w/GitHub Actions)[https://github.com/machine-learning-apps/ml-template-azure]
```
Comment (peaceiris) [inline README.md:455]:
```suggestion
- [Azure Machine Learning w/GitHub Actions](https://github.com/machine-learning-apps/ml-template-azure)
```

**Comment by sdras** [comment]
Please title-case the titles and write the whole word "with" out instead of `w/`. Thanks!

**Comment by hamelsmu** [comment]
Thanks @sdras this is ready for review

**Comment by hamelsmu** [comment]
ah nevermind I should pay attention to CI before tagging you sorry

**Comment by hamelsmu** [comment]
ok I think the linter might be catching a different error, I cannot find a place where I am misspelling GitHub in my PR :)

**Comment by sdras** [comment]
Looks like this is resolved and the linter is picking up the old error. Thank you!

**Pull request merged**